medicines/web/components/disclaimer/index.tsx

```diff
@@ -1,23 +1,31 @@
 import Link from 'next/link';
-import React, { ChangeEvent } from 'react';
+import React, { ChangeEvent, MouseEvent, useState } from 'react';
 import styled from 'styled-components';
-import { anchorColour, black, mhra } from '../../styles/colors';
+import {
+  anchorColour,
+  black,
+  mhra,
+  mhra70,
+  mhraGray,
+  mhraWhite,
+  primaryColor,
+} from '../../styles/colors';
 import { baseSpace } from '../../styles/dimensions';
 
 interface IDisclaimerProps {
-  onDisclaimerCheck: (event: ChangeEvent<HTMLInputElement>) => void;
+  onDisclaimerAgree: (event: MouseEvent<HTMLButtonElement>) => void;
   searchTerm: string;
 }
 
+const DisclaimerWrapper = styled.div`
+  border-radius: 10px;
+  background-color: rgba(254, 212, 50, 0.5);
```

Comment:
probably should be a colour constant

Reply:
This is that yellow colour on the disclaimer. I know @roughprada is working on this separatly so lets address this later